ui/src/components/executions/VarValue.vue

```diff
@@ -35,6 +36,10 @@
             itemUrl(value) {
                 return `${apiRoot}executions/${this.execution.id}/file?path=${value}`;
             },
+            download(value) {
+                const url = this.itemUrl(value);
+                window.open(url, "_blank");
+            }
```

Comment:
Already did the same somewhere and Ludo told me not to and let the default html link behaviour (meaning you encapsulate with `<a>` your button for eg.)